docs/command/mongocli-iam-projects-create.txt

```diff
@@ -32,10 +32,14 @@ Options
      - Type
      - Required
      - Description
+   * - --defaultAlertSettings
+     - 
+     - false
+     - Only for Atlas projects. Flag that indicates whether to create the new project with the default alert settings enabled. This flag defaults to false.
```

Comment:
this is also supported by cloud manager and Ops Manager 6.0 in a future, let's please add support for it

Reply:
mh.. I don't see this option for cloud manager (https://docs.cloudmanager.mongodb.com/reference/api/groups/create-one-group/) or ops manager(https://docs.opsmanager.mongodb.com/current/reference/api/groups/create-one-group/) so I don't understand why we should add this flag for them

Reply:
it's in the code, for cloud manager docs are probably outdated, for ops manager this won't happen until ops manager 6.0 which is not released yet